src/globals.js

```diff
@@ -7,4 +7,19 @@ export const clearTimeout = global.clearTimeout;
 
 export const document = window && window.document;
 export const navigator = window && window.navigator;
-export const sessionStorage = window && window.sessionStorage;
+
+let defined = {
+	sessionStorage: ( function() {
+		var x = "qunit-test-string";
+		try {
+			sessionStorage.setItem( x, x );
+			sessionStorage.removeItem( x );
+			return sessionStorage;
```

Comment:
we could use something better than a try catch, something like the `in` operator as `return 'setItem' in sessionStorage && 'removeItem' in sessionStorage && sessionStorage`

This way we also don't need to catch any false alarms if sessionStorage doesn't work on storing or removing an item. All we want is to confirm the API is available.